site/src/pages/news/20220708-newsletter-4.mdx

```diff
@@ -0,0 +1,156 @@
+---
+date: 2022-07-08
+---
+
+# Armeria Newsletter vol. 4
+
+<video class="hideOnReducedMotion" src="../../images/armeria.m4v" preload="none" autoplay muted loop style="width: 282px; height: 112px;">
+  <img src="../../images/armeria.gif" loading="lazy" width="282" height="112" />
+</video>
+
+## From the devs
+
+<div class="noBullet">
+
+- ✨ We released several releases since our last newsletter with a bunch of new exciting features!
```

Comment:
According to [Amazon's writing style tips](https://medium.com/fact-of-the-day-1/amazon-writing-style-tip-2-2f77bb3bbbb0), it recommends replacing adjectives with numbers. 😆
```suggestion
- ✨ We've released 7 new versions since our last newsletter with a bunch of new exciting features!
```